iidm/iidm-test/src/main/java/com/powsybl/iidm/network/test/FourSubstationsNodeBreakerFactory.java

```diff
@@ -0,0 +1,623 @@
+        createSwitch(s1vl2, "S1VL2_F", "S1VL2_F", SwitchKind.DISCONNECTOR, false, false, false, 0, 15);
+        createSwitch(s1vl2, "S1VL2_G", "S1VL2_G", SwitchKind.DISCONNECTOR, false, false, false, 0, 8);
+        createSwitch(s1vl2, "S1VL2_H", "S1VL2_H", SwitchKind.DISCONNECTOR, false, false, true, 0, 2);
+        createSwitch(s1vl2, "S1VL2_I", "S1VL2_I", SwitchKind.DISCONNECTOR, false, false, false, 7, 0);
+        createSwitch(s1vl2, "S1VL2_J", "S1VL2_J", SwitchKind.DISCONNECTOR, false, false, false, 1, 6);
+        createSwitch(s1vl2, "S1VL2_K", "S1VL2_K", SwitchKind.DISCONNECTOR, false, false, false, 1, 19);
+        createSwitch(s1vl2, "S1VL2_L", "S1VL2_L", SwitchKind.DISCONNECTOR, false, false, false, 1, 17);
+        createSwitch(s1vl2, "S1VL2_M", "S1VL2_M", SwitchKind.DISCONNECTOR, false, false, false, 1, 21);
+        createSwitch(s1vl2, "S1VL2_N", "S1VL2_N", SwitchKind.DISCONNECTOR, false, false, false, 1, 23);
+        createSwitch(s1vl2, "S1VL2_O", "S1VL2_O", SwitchKind.DISCONNECTOR, false, false, false, 1, 25);
+        createSwitch(s1vl2, "S1VL2_P", "S1VL2_P", SwitchKind.DISCONNECTOR, false, false, false, 1, 11);
+        createSwitch(s1vl2, "S1VL2_Q", "S1VL2_Q", SwitchKind.DISCONNECTOR, false, false, false, 1, 13);
+        createSwitch(s1vl2, "S1VL2_R", "S1VL2_R", SwitchKind.DISCONNECTOR, false, false, false, 1, 15);
+        createSwitch(s1vl2, "S1VL2_S", "S1VL2_S", SwitchKind.DISCONNECTOR, false, false, false, 1, 8);
+        createSwitch(s1vl2, "S1VL2_T", "S1VL2_T", SwitchKind.DISCONNECTOR, false, false, true, 1, 2);
+        createSwitch(s1vl2, "S1VL2_U", "S1VL2_U", SwitchKind.BREAKER, true, true, true, 2, 3);
+        createSwitch(s1vl2, "S1VL2_V", "S1VL2_V", SwitchKind.BREAKER, true, false, false, 3, 4);
+        createSwitch(s1vl2, "S1VL2_W", "S1VL2_W", SwitchKind.DISCONNECTOR, false, false, false, 3, 5);
+        createSwitch(s1vl2, "S1VL2_X", "S1VL2_X", SwitchKind.DISCONNECTOR, false, false, false, 9, 3);
+        createSwitch(s1vl2, "S1VL2_Y", "S1VL2_Y", SwitchKind.BREAKER, true, false, false, 6, 7);
+        createSwitch(s1vl2, "S1VL2_Z", "S1VL2_Z", SwitchKind.BREAKER, true, false, false, 8, 9);
+        createSwitch(s1vl2, "S1VL2_AA", "S1VL2_AA", SwitchKind.DISCONNECTOR, false, false, false, 9, 10);
+        createSwitch(s1vl2, "S1VL2_AB", "S1VL2_AB", SwitchKind.BREAKER, true, false, false, 11, 12);
+        createSwitch(s1vl2, "S1VL2_AC", "S1VL2_AC", SwitchKind.BREAKER, true, false, false, 13, 14);
+        createSwitch(s1vl2, "S1VL2_AD", "S1VL2_AD", SwitchKind.BREAKER, true, false, false, 15, 16);
+        createSwitch(s1vl2, "S1VL2_AE", "S1VL2_AE", SwitchKind.BREAKER, true, false, false, 17, 18);
+        createSwitch(s1vl2, "S1VL2_AF", "S1VL2_AF", SwitchKind.BREAKER, true, false, false, 0, 20);
+        createSwitch(s1vl2, "S1VL2_AG", "S1VL2_AG", SwitchKind.BREAKER, true, false, false, 0, 22);
+        createSwitch(s1vl2, "S1VL2_AH", "S1VL2_AH", SwitchKind.BREAKER, false, false, false, 0, 24);
```

Comment:
- S1VL2_AH should be retained.
- I think it could be better if some disconnector are open. Equipments should not be connected to both busbar section. Edit your schema to change the black circles (connected) to white circles (disconnected)
- I think you should add disconnectors for your coupler on the schema 0-7 and 1-6
- I think there is a mistake for S1VL2_AF, S1VL2_AG and S1VL2_AH: should be 19-20, 21-22 and 23-24?
- The topology close to LD2 is really strange, and not really realistic.
- Maybe you should add a breaker for LCC1
- There is an inconsistency for the switch 0-25: it doesn't exist in the code, but it exists on the schema.